src/pages/settings/PasswordPage.js

```diff
@@ -1,25 +1,158 @@
+                            title="Change Password"
+                            shouldShowBackButton
+                            onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                            onCloseButtonPress={Navigation.dismissModal}
+                        />
+                        <View style={[styles.p5, styles.flex1]}>
+                            <Text style={[styles.mb6, styles.textP]}>
+                                Changing your password will update your password for both your Expensify.com
+                                and Expensify.cash accounts.
+                            </Text>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Current Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.currentPassword}
+                                    onChangeText={currentPassword => this.setState({currentPassword})}
+                                />
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>New Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.newPassword}
+                                    onChangeText={newPassword => this.setState({newPassword})}
+                                />
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Confirm New Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.confirmNewPassword}
+                                    onChangeText={confirmNewPassword => this.setState({confirmNewPassword})}
+                                />
+                            </View>
+                            {this.props.account.requiresTwoFactorAuth ? (
+                                <View style={styles.mb6}>
+                                    <Text style={[styles.mb1, styles.formLabel]}>Two Factor Code</Text>
```

Comment:
Since we only show that input when 2FA is required, should it also have a `*` at the end? Or alternatively, maybe we can skip the `*` everywhere since every field is required?

Reply:
Resolving this since we're no longer requiring 2FA for this view.